server/routes/createGoal/createGoalController.ts

```diff
@@ -54,6 +54,36 @@ export default class CreateGoalController {
     return res.render('pages/goal/create/index', { ...view.renderArgs })
   }
 
+  // TODO: RR-734 - Create controller and view classes new create goal journey
+  newGetCreateGoalView: RequestHandler = async (req, res, next): Promise<void> => {
+    const { prisonNumber } = req.params
+    const { prisonerSummary } = req.session
+
+    req.session.newGoals = req.session.newGoals || []
```

Comment:
OK; so you're assuming we'll be using the existing form objects 🤔 
Thats a fair working assumption at the moment, but we do have a subtask around defining the form object(s), so this might change
Leave it as it is for the time being - it's OK as a skeleton, but we might change this as and when we define the form objects